docs/configuration.md

```diff
@@ -78,7 +78,7 @@ Any flag that has a value that is a comma separated list on the command line bec
 
 #### Gossip Port
 
-* Description: Port to which Pilosa should bind for internal communication.
+* Description: Port to which Pilosa should bind for internal communication. Each node in the cluster must use a different gossip port.
```

Comment:
What you added here is not necessarily the case. Each node only has to use a different gossip port if you're running all the nodes on the same machine.